packages/api/tests/endpoints/test-granules.js

```diff
@@ -613,40 +829,17 @@ test('DELETE deleting an existing unpublished granule', async (t) => {
   const { detail } = response.body;
   t.is(detail, 'Record deleted');
 
-  // verify the files are deleted
-  /* eslint-disable no-await-in-loop */
-  for (let i = 0; i < newGranule.files.length; i += 1) {
-    const file = newGranule.files[i];
-    t.false(await fileExists(file.bucket, file.key));
-  }
-  /* eslint-enable no-await-in-loop */
-
-  await deleteBuckets([
-    buckets.protected.name,
-    buckets.public.name,
-  ]);
-});
-
-test('DELETE for a granule with a file not present in S3 succeeds', async (t) => {
```

Comment:
Was the removal of this test intentional?

Reply:
I see. I think this test moved to being a test of `deleteGranulesAndFiles`, which makes sense

Reply:
Yeah, apologies. There are a lot of tests moving around in this last batch of commits.